docker/README.md

```diff
@@ -30,6 +30,27 @@ logging:
 ```
 Mind the indent. Explanations for all the logging drivers in `docker-compose` can be found [here](https://docs.docker.com/config/containers/logging/local/).
 
+## Run the integration tests
+```
+docker-compose -f docker-compose.yml -f integration-test.yml up --exit-code-from sidechain-integration-test
+```
+
+## Run the fork simulator
+Build the docker-compose setup with
+```
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml -f fork-inducer.yml -f integration-test.yml build
+```
 
+This requires the docker BuildKit (docker version >= 18.09) and support for it in docker-compose (version >= 1.25.0)
 
+Run the 2-worker setup with a fork inducer (pumba) that delays the traffic on worker 2
+```
+docker-compose -f docker-compose.yml -f fork-inducer.yml -f integration-test.yml up --exit-code-from sidechain-integration-test
+```
+
+This should show that the integration test fails, because we had an unhandled fork in the sidechain. Clean up the containers after each run with:
+```
+docker-compose -f docker-compose.yml -f fork-inducer.yml -f integration-test.yml down
+```
 
+We need these different compose files to separate the services that we're using. E.g. we want the integration test and fork simulator to be optional. The same could be solved using `profiles` - but that requires a more up-to-date version of `docker-compose`.
```

Comment:
Very nice! There are so many ports both, inside and outside the docker container. I think a table with all default ports would be very helpful.

Reply:
Yes, indeed. We have a diagram for that in the integritee book: https://book.integritee.network/sidechain_system_networking.html

Reply:
Thats helpful. What I meant is a table with the default values of the ports used in this setup. Something like

```
|                     | internal | outside |
|---------------------|----------|---------|
| Node port           | 9912     | 9912    |
| Worker1 worker port | 2011     | 2011    |
| Worker1 http port   | 4645     | 4645    |
| Worker2 worker port | 2012     | 2012    |
| Worker2 http port   | 4646     | 4646    |
```

Reply:
I'm not sure if I see the benefit? If you're interested in the specific port setup, look at the compose file (and it shouldn't be relevant for most cases, since you just want to run it). This would be documentation that very easily goes out-of-sync with the actual values.